doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+          "type": "boolean"
+        },
+        "showTerminalTitleInTitlebar": {
+          "default": true,
+          "description": "When set to true, titlebar displays the title of the selected tab. When set to false, titlebar displays 'Windows Terminal'.",
+          "type": "boolean"
+        },
+        "wordDelimiters": {
+          "default": " ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}~?│",
+          "description": "Determines the delimiters used in a double click selection.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "alwaysShowTabs",
+        "defaultProfile",
+        "initialCols",
+        "initialRows",
+        "requestedTheme",
+        "showTerminalTitleInTitlebar"
+      ],
+      "type": "object"
+    },
+    "profiles": {
+      "description": "Properties are specific to each unique profile.",
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "acrylicOpacity": {
+            "default": 0.5,
+            "description": "When useAcrylic is set to true, it sets the transparency of the window for the profile. Accepts floating point values from 0-1 (default 0.5).",
+            "maximum": 1,
+            "minimum": 0,
+            "type": "number"
+          },
+          "background": {
+            "$ref": "#/definitions/Color",
+            "description": "Sets the background color of the profile. Overrides background set in color scheme if colorscheme is set. Uses hex color format: \"#rrggbb\". Default \"#000000\" (black)."
+          },
+          "backgroundImage": {
+            "description": "(Not in SettingsSchema.md)",
+            "type": "string"
+          },
+          "backgroundImageAlignment": {
+            "default": "center",
+            "enum": [
+              "bottom",
+              "bottomLeft",
+              "bottomRight",
+              "center",
+              "left",
+              "right",
+              "top",
+              "topLeft",
+              "topRight"
+            ],
+            "type": "string"
+          },
+          "backgroundImageOpacity": {
+            "description": "(Not in SettingsSchema.md)",
+            "maximum": 1,
+            "minimum": 0,
+            "type": "number"
+          },
+          "backgroundImageStretchMode": {
+            "description": "(Not in SettingsSchema.md) Possible values: \"none\", \"fill\", \"uniform\", \"uniformToFill\" (default)",
+            "enum": [
+              "fill",
+              "none",
+              "uniform",
+              "uniformToFill"
+            ],
+            "type": "string"
+          },
+          "closeOnExit": {
+            "default": true,
+            "description": "When set to true (default), the selected tab closes when exit is typed. When set to false, the tab will remain open when exit is typed.",
```

Comment:
not just when exit is typed, but when the connected application exits in any manner.